docs/components/modeler/web-modeler/token-simulation.md

```diff
@@ -1,13 +1,25 @@
 ---
 id: token-simulation
 title: Token simulation
-description: You can use the token simulation feature to observe how the process will behave when it is executed.
+description: Use token simulation to get a deeper understanding of the BPMN execution semantics by playing, pausing, and stepping through your processes.
```

Comment:
> by playing, pausing, and stepping through your processes

@christinaausley Could you elaborate what this contributes to the description?

Reply:
Correct!